$Id$
Installation as root is never done anyway, so remove this crap
--- ltp-full-20061121.orig/Makefile	2006-11-21 21:31:20.000000000 +0100
+++ ltp-full-20061121/Makefile	2006-12-06 12:08:02.000000000 +0100
@@ -44,15 +44,10 @@ all: libltp.a 
 	@$(MAKE) -C pan $@
 	@$(MAKE) -C testcases $@
 	@$(MAKE) -C tools $@
-	@echo
-	@echo "***********************************************"
-	@echo "** You now need to do a make install as root **"
-	@echo "***********************************************"
 
 install: all
 	@$(MAKE) -C testcases install
 	@$(MAKE) -C tools install
-	@./IDcheck.sh
 
 libltp.a:
 	@$(MAKE) -C lib $@
